src/main/java/kr/codesquad/secondhand/application/auth/NaverRequester.java

```diff
@@ -18,12 +18,18 @@
 import org.springframework.util.MultiValueMap;
 import org.springframework.web.client.RestTemplate;
 
-@RequiredArgsConstructor
 @Component
 public class NaverRequester {
 
     private final RestTemplate restTemplate;
     private final OauthProvider oauthProvider;
+    private final String DEFAULT_PROFILE_IMAGE;
+
+    public NaverRequester(RestTemplate restTemplate, OauthProvider oauthProvider, @Value("${custom.default-profile}") String DEFAULT_PROFILE_IMAGE) {
```

Comment:
`@Value`👍 파라미터 이름은 `defaultProfileImage`로 해도 괜찮을 것 같아요~!
```suggestion
    public NaverRequester(RestTemplate restTemplate, OauthProvider oauthProvider, @Value("${custom.default-profile}") String defaultProfileImage) {
```